changelog.rst

```diff
@@ -13,6 +13,17 @@ documentation.
     Do you also want to participate in the Symfony Documentation? Take a look
     at the ":doc:`/contributing/documentation/overview`" article.
 
+November, 2016
```

Comment:
The changelogs are automatically generated based on the merge commits. Please remove the changes in this file.